frontend/www/js/omegaup/arena/clarifications.ts

```diff
@@ -1,34 +1,51 @@
-import Vue from 'vue';
 import * as api from '../api';
 import * as time from '../time';
 import { types } from '../api_types';
+import clarificationStore from './clarificationsStore';
 
 export interface ClarificationEvent {
```

Comment:
esto ya no es un evento!

```suggestion
export interface ContestClarification {
```

Reply:
Listo